osmosis/assetlist.json

```diff
@@ -15225,6 +15225,34 @@
         }
       ]
     },
+    {
+      "description": "ashLAB - Burned LAB",
+      "extended_description": "ashLAB - reciept token recieved when burning LAB via ASH DAOs Furnace",
+      "denom_units": [
+    {
+        "denom": "factory/osmo1svj5kd8kzj7xxtrd6ftjk0856ffpyj4egz7f9pd9dge5wr4kwansmefq07/lab.ash",
+        "exponent": 0
+    },
+    {
+        "denom": "ashLAB",
+        "exponent": 6
+    }
+  ],
+      "type_asset": "sdk.coin",
+      "address": "osmo1svj5kd8kzj7xxtrd6ftjk0856ffpyj4egz7f9pd9dge5wr4kwansmefq07",
+      "base": "factory/osmo1svj5kd8kzj7xxtrd6ftjk0856ffpyj4egz7f9pd9dge5wr4kwansmefq07/lab.ash",
+      "name": "Burned LAB",
+      "display": "ashLAB",
+      "symbol": "ashLAB",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/ashLAB.png"
+  },
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/ashLAB.png"
+    }
+  ]
+},
```

Comment:
Please try to fix the formatting here (number of spaces). And relocate the asset to the bottom of the list.